### ISSUE TITLE
The kitchen vending machine doesn't have spoons.

### ISSUE THREAD

**Issue by moonheart08** [space-wizards/space-station-14#11156]
## Description
~~It's like a fork but for soup. (There's no spoon item but we do have forks, we should probably have both)~~
We apparently have spoons but they're unobtainable.

**Comment by biometricPsychography** [comment]
![giphy](https://user-images.githubusercontent.com/14880397/189671909-2a2aa71f-8c03-4771-8f7d-5e7107b713e9.gif)

**Comment by EmoGarbage404** [comment]
Replace forks with sporks so we can blatantly copy behavior.

**Comment by locdog21** [comment]
It looks like spoons and forks were added in 74a4cc63cc6627fa3c7f21c28e60f20ba4e97ae3, but when I spawn them into the game, I can use neither spoon, nor fork, to eat apple cake, or enchiladas. These are the only two food items I used them on, but I'd guess that the spoon and fork mechanics are not implemented yet.
I think we should ensure that the mechanics are in place prior to being made obtainable in-game.
